sdk/servicebus/Azure.Messaging.ServiceBus/src/Management/QueueDescriptionExtensions.cs

```diff
@@ -210,13 +210,10 @@ public static void NormalizeDescription(this QueueDescription description, strin
 
         private static string NormalizeForwardToAddress(string forwardTo, string baseAddress)
         {
-            if (!Uri.TryCreate(forwardTo, UriKind.Absolute, out var forwardToUri))
-            {
-                if (!baseAddress.EndsWith("/", StringComparison.OrdinalIgnoreCase))
-                {
-                    baseAddress += "/";
-                }
+            baseAddress = new UriBuilder(baseAddress).Uri.ToString();
```

Comment:
Why are we changing the semantics here?

Reply:
In track-1 we use endpoint - `sb://testservicebus.servicebus.windows.net` and in track-2 we use fully qualified namespace - `testservicebus.servicebus.windows.net`. The above code basically changes the fully qualified namespace to uri - `http://testservicebus.servicebus.windows.net/`.

Reply:
Ah ok